cmd/rpcdaemon/commands/trace_api.go

```diff
@@ -16,17 +16,17 @@ import (
 // TraceAPI RPC interface into tracing API
 type TraceAPI interface {
 	// Ad-hoc (see ./trace_adhoc.go)
-	ReplayBlockTransactions(ctx context.Context, blockNr rpc.BlockNumberOrHash, traceTypes []string) ([]*TraceCallResult, error)
-	ReplayTransaction(ctx context.Context, txHash libcommon.Hash, traceTypes []string) (*TraceCallResult, error)
+	ReplayBlockTransactions(ctx context.Context, blockNr rpc.BlockNumberOrHash, traceTypes []string, gasBailOut *bool) ([]*TraceCallResult, error)
+	ReplayTransaction(ctx context.Context, txHash libcommon.Hash, traceTypes []string, gasBailOut *bool) (*TraceCallResult, error)
 	Call(ctx context.Context, call TraceCallParam, types []string, blockNr *rpc.BlockNumberOrHash) (*TraceCallResult, error)
 	CallMany(ctx context.Context, calls json.RawMessage, blockNr *rpc.BlockNumberOrHash) ([]*TraceCallResult, error)
 	RawTransaction(ctx context.Context, txHash libcommon.Hash, traceTypes []string) ([]interface{}, error)
 
 	// Filtering (see ./trace_filtering.go)
-	Transaction(ctx context.Context, txHash libcommon.Hash) (ParityTraces, error)
-	Get(ctx context.Context, txHash libcommon.Hash, txIndicies []hexutil.Uint64) (*ParityTrace, error)
-	Block(ctx context.Context, blockNr rpc.BlockNumber) (ParityTraces, error)
-	Filter(ctx context.Context, req TraceFilterRequest, stream *jsoniter.Stream) error
+	Transaction(ctx context.Context, txHash libcommon.Hash, gasBailOut *bool) (ParityTraces, error)
```

Comment:
`{"jsonrpc":"2.0", "method":"trace_block","params":["0x258b00c", true],"id":0}` and `{"jsonrpc":"2.0", "method":"trace_block","params":["0x258b00c"],"id":0}` should both work.
That's the reason of defining `gasBailout` as a pointer.